util/convert_wycheproof/convert_wycheproof.go

```diff
@@ -224,6 +224,8 @@ var defaultInputs = []string{
 	"hmac_sha256_test.json",
 	"hmac_sha384_test.json",
 	"hmac_sha512_test.json",
+	"hmac_sha512_224_test.json",
+	"hmac_sha512_256_test.json",
```

Comment:
HMAC w/ SHA512-256 wasn't previously tested here, so I added the wycheproof KAT files from [their github repository](https://github.com/google/wycheproof).